helix-term/src/ui/prompt.rs

```diff
@@ -459,8 +459,8 @@ impl Component for Prompt {
                 (self.callback_fn)(cx, &self.line, PromptEvent::Abort);
                 return close_fn;
             }
-            alt!('b') | alt!(Left) => self.move_cursor(Movement::BackwardWord(1)),
-            alt!('f') | alt!(Right) => self.move_cursor(Movement::ForwardWord(1)),
+            alt!('b') | alt!(Left) | ctrl!(Left) => self.move_cursor(Movement::BackwardWord(1)),
+            alt!('f') | alt!(Right) | ctrl!(Right) => self.move_cursor(Movement::ForwardWord(1)),
```

Comment:
```suggestion
            alt!('b') | ctrl!(Left) => self.move_cursor(Movement::BackwardWord(1)),
            alt!('f') | ctrl!(Right) => self.move_cursor(Movement::ForwardWord(1)),
```

The docs should also be updated, this will make it similar to insert mode, I forgot about this when I did the changes for insert mode, ctrl is indeed better keys given that it is behaves similarly to many other editors (and keymap).